pkg/server/router/submission.go

```diff
@@ -0,0 +1,125 @@
+package router
+
+import (
+	"context"
+	"fmt"
+	"github.com/TBD54566975/ssi-sdk/credential/exchange"
+	"github.com/pkg/errors"
+	"github.com/tbd54566975/ssi-service/internal/keyaccess"
+	"github.com/tbd54566975/ssi-service/pkg/server/framework"
+	svcframework "github.com/tbd54566975/ssi-service/pkg/service/framework"
+	"github.com/tbd54566975/ssi-service/pkg/service/submission"
+	"net/http"
+)
+
+type SubmissionRouter struct {
+	service *submission.Service
+}
+
+func NewSubmissionRouter(s svcframework.Service) (*SubmissionRouter, error) {
+	if s == nil {
+		return nil, errors.New("service cannot be nil")
+	}
+	service, ok := s.(*submission.Service)
+	if !ok {
+		return nil, fmt.Errorf("could not create submission router with service type: %s", s.Type())
+	}
+	return &SubmissionRouter{service: service}, nil
+}
+
+type CreateSubmissionRequest struct {
+	PresentationJwt keyaccess.JWT `json:"presentationJwt" validate:"required"`
+}
+
+type Operation struct {
+	ID     string          `json:"id"`
+	Done   bool            `json:"bool"`
+	Result OperationResult `json:"result"`
+}
+
+type OperationResult struct {
+	Error    string                          `json:"error"`
+	Response exchange.PresentationSubmission `json:"response"`
+}
+
+// CreateSubmission godoc
+// @Summary      Create Submission
+// @Description  Creates a submission in this server ready to be reviewed.
+// @Tags         SubmissionAPI
+// @Accept       json
+// @Produce      json
+// @Param        request  body      CreateSubmissionRequest  true  "request body"
+// @Success      201      {object}  Operation
+// @Failure      400      {string}  string  "Bad request"
+// @Failure      500      {string}  string  "Internal server error"
+// @Router       /v1/presentations/submissions [put]
+func (sr SubmissionRouter) CreateSubmission(ctx context.Context, w http.ResponseWriter, r *http.Request) error {
+	resp := Operation{}
```

Comment:
`var resp Operation`

https://github.com/uber-go/guide/blob/master/style.md#use-var-for-zero-value-structs

note - don't think our README points to this style doc even though it should. may need to add it

Reply:
ty - feel free to add to the README